lively.morphic/svg-morph/svg-morph.js

```diff
@@ -68,20 +68,89 @@ export class SVGMorph extends Morph {
       fill: { defaultValue: Color.transparent },
       borderColor: { defaultValue: Color.transparent },
       svgUrl: {
+        defaultValue: ''
```

Comment:
do we need a default value, if it is empty anyways?

Reply:
Potentially we want to save a default path in there when an empty svg morph is created? But can be removed for now